src/test/ui/lint/command-line-register-unknown-lint-tool.rs

```diff
@@ -0,0 +1,4 @@
+// compile-flags: -A unknown_tool::foo
+// check-fail
```

Comment:
This comment doesn't do anything, tests are check-fail by default.

Reply:
Thanks, the error-pattern suggestion worked!